### PR TITLE
Remove example code from challenge seed

### LINKED ISSUE #39081: Remove example code from seed/starter code
Challenge: https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/generate-random-whole-numbers-with-javascript

Forum post: https://www.freecodecamp.org/forum/t/problems-with-generate-random-whole-numbers-with-javascript/404092

I think we should remove the example code from the seed code. We already have an example in the challenge text. Right now, it's just some odd unused code at the beginning of the challenge.

If we still want the example code it should be moved to the challenge text instead.

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #39081

Per the linked issue, I've removed the example code from the challenge seed. The example code is already present in the challenge instructions (without the variable declaration).

### CODE REVIEW

**Review by ShaunSHamilton — DISMISSED** [review]
<s>LGTM 👍 </s>
Ignore this. The tests need to be changed as well, @nhcarrigan . The tests are taking into account the extra line of code you removed...

**Comment by nhcarrigan** [comment]
Ah, thank you! I was trying to figure out why that test was failing.
I just stepped out so I won't be able to fix this until later this evening. 🙂

**Comment by nhcarrigan** [comment]
@Sky020 Okay I fixed the tests. 
Took a good 30 minutes to figure out why my first fix wasn't working, only to realise it was a different test failing. 😅 Should be good to go now!

**Review by lasjorg — DISMISSED** [review]
I guess it's fine to allow for more than one Math.random/Math.floor in the code (`>=`).

Tested in Gitpod, LGTM.

---

Just for future reference.

1. Please test your code locally when making changes to the test code. Make sure the tests pass before pushing the code.

2. Prefer creating a new branch for your PRs. It's better not to make changes to your master branch (read: [Making changes locally](https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally?id=making-changes-locally)).

**Comment by nhcarrigan** [comment]
Ah, thank you! Those are good tips and I will take them to heart. 🙂 
My thought process with the `>=` was to allow for possible instances where the user had written an extra line or did not clean up their code.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/generate-random-whole-numbers-with-javascript.english.md [hunk 1]
```diff
@@ -30,11 +30,11 @@ tests:
   - text: The result of <code>randomWholeNum</code> should be a whole number.
     testString: assert(typeof randomWholeNum() === "number" && (function(){var r = randomWholeNum();return Math.floor(r) === r;})());
   - text: You should be using <code>Math.random</code> to generate a random number.
```
Comment (thecodingaviator) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/generate-random-whole-numbers-with-javascript.english.md:32]:
```suggestion
  - text: You should use <code>Math.random</code> to generate a random number.
```
To maintain consistency with what we say in the last test

**Review by thecodingaviator — CHANGES_REQUESTED** [review]

**Review by thecodingaviator — APPROVED** [review]

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍

**Pull request merged**